tests/unit/test_user.py

```diff
@@ -0,0 +1,63 @@
+from fastapi.testclient import TestClient
+from sqlmodel import Session
+
+from habits.models import User
+
+
+def test_create_user(client: TestClient):
+    response = client.post("/users/", json={"name": "craig", "handle": "greywidget"})
+    data = response.json()
+
+    assert response.status_code == 200
```

Comment:
Not sure why docs have 200 here while 201 is more accurate for post create, do you know?

Reply:
Good question, not sure.

I couldn't see any specific mention of this in the docs.
My working assumption is that Sebastián built it so that 200 is the default for everything and you just add the status_code parameter in the route decorator if you want something other.

I did find some documentation for chaning the default, if for example you want to have an endpoint that can CREATE or UPDATE: 
https://fastapi.tiangolo.com/advanced/response-change-status-code/

I've changed my 2 POSTS to specify `status_code=status.HTTP_201_CREATED` and also explicitly defined the 400 and 404 errors that way

I did hunt around for a while in the routes and decorator code to try to see where the default of 200 was actually being defined but I failed to find it...

Reply:
Thanks for the research, no worries for now, let's build the next features so we can ship and use it :)